### PR TITLE
Shf

### PR DESCRIPTION
**Pull request by serhiishf**
Add table with books list on create training page

### CODE REVIEW

File: src/components/Library/LibraryBook/DoneEl/DoneEl.tsx [hunk 1]
```diff
@@ -36,7 +36,7 @@ const DoneEl: FC<Props> = ({ _id, rating, resume, onUpdate }) => {
             <ModalResume
               bookId={_id}
               rating={rating}
-              resume={resume}
+              resume={resume ? resume : ''}
```
Comment (MarinaTripetska) [inline src/components/Library/LibraryBook/DoneEl/DoneEl.tsx]: Please, cancel it. On the backend is only string type/ I will change it  in Trello.

File: src/components/Library/LibraryBook/LibraryBook.tsx [hunk 2]
```diff
@@ -65,7 +65,7 @@ const LibraryBook: React.FC<BookProps> = ({ book, update, onDelete }) => {
           <DoneEl
             _id={book._id}
             rating={book.rating}
-            resume={book.resume}
+            resume={book.resume? book.resume : ''}
```
Comment (MarinaTripetska) [inline src/components/Library/LibraryBook/LibraryBook.tsx]: Please, cancel it. On the backend is only string type/ I will change it in Trello.

File: src/services/books/books-service.ts [hunk 3]
```diff
@@ -42,7 +42,7 @@ export interface Book {
     name: string;
     email: string;
   };
-  resume: string;
+  resume: string | null;
```
Comment (MarinaTripetska) [inline src/services/books/books-service.ts]: Please, cancel it. On the backend is only string type/ I will change it in Trello.

**Review by MarinaTripetska — CHANGES_REQUESTED** [review]

**Review by Katerina-Zamiatina — APPROVED** [review]

File: src/components/Library/LibraryBook/DoneEl/DoneEl.tsx (hunk 1 above)
Comment (serhiishf) [reply src/components/Library/LibraryBook/DoneEl/DoneEl.tsx]: Fixed

File: src/services/books/books-service.ts (hunk 3 above)
Comment (serhiishf) [reply src/services/books/books-service.ts]: Fixed

File: src/components/Library/LibraryBook/LibraryBook.tsx (hunk 2 above)
Comment (serhiishf) [reply src/components/Library/LibraryBook/LibraryBook.tsx]: Fixed

**Review by Katerina-Zamiatina — APPROVED** [review]

**Review by MarinaTripetska — APPROVED** [review]

**Pull request merged**